puma/puma.init.erb

```diff
@@ -1,18 +0,0 @@
-description "<%=user%> puma"
-
-start on (filesystem and static-network-up)
-stop on runlevel [!2345]
-
-respawn
-
-kill signal QUIT
-
-chdir <%=app_root%>
-setuid <%=user%>
-console log
-env LANG=en_AU.UTF-8
-env RACK_ENV=<%=env%>
-
-script
-  ./bin/puma
-end script
```

Comment:
Yes, I have tested it on Rackspace. When I initially created the puma upstart template I just cargoculted the unicorn one without giving it much thought. I don't think it's necessary to have another template that doesn't do anything special.